### PR TITLE
LOAD CSV warnings

### PR DESCRIPTION
**Pull request by henriknyman**
- Warn when a LOAD CSV is followed by an Eager 
- Warn when LOAD CSV plus MATCH/MERGE on large non-indexed label

### CODE REVIEW

File: community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/executionplan/ExecutionPlanBuilder.scala [hunk 1]
```diff
@@ -142,13 +143,40 @@ class ExecutionPlanBuilder(graph: GraphDatabaseService, statsDivergenceThreshold
       def isPeriodicCommit: Boolean = compiledPlan.periodicCommit.isDefined
 
       def runtimeUsed = CompiledRuntimeName
+
+      def notifications = Seq.empty
+    }
+  }
+
+
+  def checkForNotifications(pipe: Pipe): Seq[InternalNotification] = {
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/executionplan/ExecutionPlanBuilder.scala]: private

**Comment by jexp** [comment]
I can confirm that it generates the warning.

**Comment by jexp** [comment]
Probably split into two PRs 
The first one was already ok and could have been merged :)

**Comment by pontusmelke** [comment]
@jexp Yeah we should probably, the second one depends on the first though

**Pull request merged**